mobile_app_connector/models/app_hub.py

```diff
@@ -262,7 +262,7 @@ def _assign_order(self, messages):
         for tile in to_order:
             recent_group = recent_content[tile['SubType']]
             if recent_group['max_number_tile'] > len(recent_group['tiles']) \
-                    and (tile['SubType'] != 'LE_T1' or tile['UnReadRecently']):
+                    and (tile['SubType'] != 'LE_T1' or tile.get('UnReadRecently', False)):
```

Comment:
line too long. Note that returning False is the default behaviour of the get method.
```suggestion
                    and (tile['SubType'] != 'LE_T1' or tile.get('UnReadRecently')):
```